### DISCUSSION TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'accounts'

### DISCUSSION THREAD

**Discussion by desdemona462** [anasty17/mirror-leech-telegram-bot#1229] [Q&A]
app_1  | 2022-11-19 05:35:20,613 - root - INFO - Creating client from BOT_TOKEN
app_1  | 2022-11-19 05:35:20,615 - root - WARNING - MEGA API KEY not provided!
app_1  | 2022-11-19 05:35:20,615 - root - WARNING - MEGA Credentials not provide                                                                             d!
app_1  | 2022-11-19 05:35:20,615 - root - WARNING - BASE_URL not provided!
app_1  | chmod: cannot access 'accounts': No such file or directory
app_1  | 2022-11-19 05:35:21,310 - root - INFO - Initializing Aria2c
app_1  | 2022-11-19 05:35:23,302 - bot - INFO - Creating Telegraph Account
app_1  | 2022-11-19 05:35:27,457 - apscheduler.scheduler - INFO - Scheduler star                                                                             ted
app_1  | 2022-11-19 05:35:27,772 - bot - INFO - Bot Started!
app_1  | 2022-11-19 05:35:36,701 - bot - INFO - listing: tbs
app_1  | Exception in thread Thread-3:
app_1  | Traceback (most recent call last):
app_1  |   File "/usr/lib/python3.8/threading.py", line 932, in _bootstrap_inner
app_1  |     self.run()
app_1  |   File "/usr/lib/python3.8/threading.py", line 870, in run
app_1  |     self._target(*self._args, **self._kwargs)
app_1  |   File "/usr/src/app/bot/modules/list.py", line 42, in _list_drive
app_1  |     gdrive = GoogleDriveHelper()
app_1  |   File "/usr/src/app/bot/helper/mirror_utils/upload_utils/gdriveTools.p                                                                             y", line 60, in __init__
app_1  |     self.__service = self.__authorize()
app_1  |   File "/usr/src/app/bot/helper/mirror_utils/upload_utils/gdriveTools.p                                                                             y", line 82, in __authorize
app_1  |     globals()['SERVICE_ACCOUNTS_NUMBER'] = len(listdir("accounts"))
app_1  | FileNotFoundError: [Errno 2] No such file or directory: 'accounts'

**HOW TO FIX THIS? I have uploaded the accounts.zip & token.pickle in the mirrorbot directory**

**Comment by anasty17** [comment]
Read readme

**Reply by desdemona462** [reply]
tell one thing

using accounts.zip and token.pickle links via config.env variables not possible now? Because there are not such variable for accounts.zip and token.pickle

**Reply by anasty17** [reply]
Join official channel on tg and read.
